src/web/modules/gateway_accounts/gateway_accounts.http.ts

```diff
@@ -73,16 +74,28 @@ async function listCSV(req: Request, res: Response): Promise<void> {
 
 async function create(req: Request, res: Response): Promise<void> {
   const serviceId = req.query.service as string
+  const live = req.query.live as string
+  const provider = req.query.provider as string
+
+  if (!live || !provider) {
+    throw new Error('Expected "live" and "provider" query parameters')
+  }
+
   const context: {
```

Comment:
can we move this to a separate type/interface and not include it as part of the assignment? appreciate this is most likely only used here but imo it makes code more difficult to read

Reply:
Yeh agreed it is unsightly, will do this separately.